### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by faheem102**

### CODE REVIEW

File: javascript/javascript2/week3/index.html
Comment (dinythomas89) [inline javascript/javascript2/week3/index.html:22]: better use these styling in the css file

File: javascript/javascript2/week3/index.html
Comment (dinythomas89) [inline javascript/javascript2/week3/index.html:27]: same here

File: javascript/javascript2/week3/index.js
Comment (dinythomas89) [inline javascript/javascript2/week3/index.js:15]: console.log (stringToLog) is fine here

File: javascript/javascript2/week3/index.js
Comment (dinythomas89) [inline javascript/javascript2/week3/index.js:104]: or `if(shouldTellFunnyJoke){}` works the same

**Review by dinythomas89 — APPROVED** [review]
Its not a good practice to use inline styling.

**Comment by faheem102** [comment]
hi diny 
thank you for your feedback. i have now made all the suggested changes. please have a look and if it is fine then let me know. i will merge the pull request after your approval. thank you

**Pull request merged**
